app/assets/javascripts/components/markdown-editor.js

```diff
@@ -26,6 +27,11 @@ MarkdownEditor.prototype.init = function () {
   // Handle events
   this.$previewButton.addEventListener('click', $module.boundPreviewButtonClick)
   this.$editButton.addEventListener('click', $module.boundEditButtonClick)
+
+  // Bubble focus events
+  this.bubbleFocus(this.$input, this.$module, this.$container)
```

Comment:
Could we avoid passing module here? I think this would create an opportunity for a more explanatory method name :-).